tutorials/secretstore/README.md

```diff
@@ -144,7 +144,7 @@ env:
 -->
 
 ```bash
-dapr run --app-id nodeapp --components-path ./components --app-port 3000 --dapr-http-port 3500 node app.js
```

Comment:
Is it expected that many of them kept the `components` name for folder?

Reply:
@greenie-msft suggested we just change the `component` folder name to `resources` for State Mgmt and Service Invocation for now, since those are the only quickstarts with more than just component files